apps/prairielearn/elements/pl-file-editor/info.json

```
@@ -9,5 +9,10 @@
     ],
     "elementScripts": ["pl-file-editor.js"],
     "elementStyles": ["pl-file-editor.css"]
+  },
+  "dynamicDependencies": {
+    "nodeModulesScripts": {
+      "@viz-js": "@viz-js/viz/lib/viz-standalone.js"
```

Nit, I'd rather set a convention of using the actual package name.

```suggestion
      "@viz-js/viz": "@viz-js/viz/lib/viz-standalone.js"
```

This will need to be updated in the client-side JS too.

---

Done in 05f543f2cdb4fce0802acba017a5a09de2f2936e.